openssl-sys/src/nid.rs

```diff
@@ -0,0 +1,169 @@
+#[derive(Copy, Clone)]
+#[repr(usize)]
+pub enum Nid {
```

Comment:
I'd move this into the openssl crate so people don't need to add an explicit dependency on openssl-sys to get a hold of it.